### PR TITLE
Update Firefox 108 CSS features

### PR DESCRIPTION
**Pull request by ramiy**
### Description

Update the CSS section.

- Fix wrong information - trig functions and math constants can me used in all math functions, not only inside `calc()` function.
- Mention the support of CSS `round()` function.
- Add links to MDN functions.

Related: #22153, #22114

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ This article provides information about the changes in Firefox 108 that will aff
 
 ### CSS
 
-- Trigonometric functions are enabled with the `layout.css.trig.enabled` preference set to `true` by default.
-  This allows the use of `sin()`, `cos()`, `tan()`, `asin()`, `acos()`, `atan()`, and `atan2()` functions within the CSS [`calc()`](/en-US/docs/Web/CSS/calc) function.
-  Additionally, a CSS [`<calc-constant>`](/en-US/docs/Web/CSS/calc-constant) type is implemented to allow for well-known constants such as `pi` and `e` within the `calc()` function ({{bug(1774589)}}, {{bug(1682444)}}).
-
+- [Trigonometric functions](/en-US/docs/Web/CSS/CSS_Functions#trigonometric_functions) are now enabled with the `layout.css.trig.enabled` preference set to `true` by default.
+  This allows the use of [`sin()`](/en-US/docs/Web/CSS/sin), [`cos()`](/en-US/docs/Web/CSS/cos), [`tan()`](/en-US/docs/Web/CSS/tan), [`asin()`](/en-US/docs/Web/CSS/asin), [`acos()`](/en-US/docs/Web/CSS/acos), [`atan()`](/en-US/docs/Web/CSS/atan), and [`atan2()`](/en-US/docs/Web/CSS/atan2) functions everywhere ({{bug(1774589)}}, {{bug(1787070)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/108/index.md:27]: Please revert this change, the addition of a [link to Trig functions](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions#trigonometric_functions) is sufficient.

```suggestion
  This allows the use of `sin()`, `cos()`, `tan()`, `asin()`, `acos()`, `atan()`, and `atan2()` functions ({{bug(1774589)}}, {{bug(1787070)}}).
```

Comment (ramiy) [reply files/en-us/mozilla/firefox/releases/108/index.md:27]: Why? Don't you want to let the readers explore those functions directly?

Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/108/index.md:27]: This is not a blocker, but in [the following section](https://github.com/mdn/content/pull/22367/files#diff-91352c37fdcac0cf841adcbc557ec37a915b637e374efc320bccd8fc32e60f6cL31) we do not link the individual properties, only a link to a summary document, so it would be nice to have this consistent in the page.

Also we should either elaborate what "everywhere" means or remove it.

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ This article provides information about the changes in Firefox 108 that will aff
...
+- CSS [`<calc-constant>`](/en-US/docs/Web/CSS/calc-constant) type is implemented to allow for well-known constants such as `pi` and `e` within [math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) ({{bug(1682444)}}, {{bug(1787070)}}).
+- CSS [`round()`](/en-US/docs/Web/CSS/round) function is now supported via the the `layout.css.round.enabled` preference, which is set to `false` by default ({{bug(1764850)}}).
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/108/index.md:29]: @ramiy I'm removing this entry - we only document released features here, which are _by definition_ those features that are not behind a preference.

@bsmth is correct though, we need to make sure this is recorded in the experimental features page.

```suggestion
```

**Comment by hamishwillee** [comment]
I've removed the `round()` mention as this is not released. The remainder are for @ramiy and @bsmth to resolve together.

File: files/en-us/mozilla/firefox/releases/108/index.md (hunk 1 above)
Comment (ramiy) [reply files/en-us/mozilla/firefox/releases/108/index.md:27]: Not sure about this approach bug ok, I'll remove the links to all the trig functions.

The "everywhere" part is from - https://bugzilla.mozilla.org/show_bug.cgi?id=1787070

Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/108/index.md]: Hi Ramiy, thanks. In the case of the Fx bug "everywhere" means 'in every Firefox version' [not just nightly](https://phabricator.services.mozilla.com/rMOZILLACENTRAL7068629c061862fbe926d3c2d6694878a8b43b23) so it's best to remove this from the release note.

**Review by bsmth — APPROVED** [review]
Thanks @ramiy 🙌🏻
